Demos/Blazorise.Demo/Components/TopMenu.razor

```diff
@@ -165,6 +172,97 @@
+        { "tests/utilities/grid", "Pages/Tests/GridPage.razor" },
+        { "tests/icons", "Pages/Tests/IconsPage.razor" },
+        { "tests/jumbotrons", "Pages/Tests/JumbotronsPage.razor" },
+        { "tests/utilities/layout", "Pages/Tests/LayoutPage.razor" },
+        { "tests/listgroup", "Pages/Tests/ListGroupPage.razor" },
+        { "tests/listview", "Pages/Tests/ListViewPage.razor" },
+        { "tests/livecharts", "Pages/Tests/LiveChartsPage.razor" },
+        { "tests/loadingindicator", "Pages/Tests/LoadingIndicatorPage.razor" },
+        { "tests/lottie-animation", "Pages/Tests/LottieAnimationPage.razor" },
+        { "tests/markdown", "Pages/Tests/MarkdownPage.razor" },
+        { "tests/misc-forms", "Pages/Tests/MiscFormPage.razor" },
+        { "tests/modal-provider", "Pages/Tests/ModalProviderPage.razor" },
+        { "tests/modals", "Pages/Tests/ModalsPage.razor" },
+        { "tests/offcanvas", "Pages/Tests/OffcanvasPage.razor" },
+        { "tests/utilities/overflow", "Pages/Tests/OverflowPage.razor" },
+        { "tests/paginations", "Pages/Tests/PaginationsPage.razor" },
+        { "tests/pickers", "Pages/Tests/PickersPage.razor" },
+        { "tests/utilities/position", "Pages/Tests/PositionsPage.razor" },
+        { "tests/progressbars", "Pages/Tests/ProgressBarsPage.razor" },
+        { "tests/qrcode", "Pages/Tests/QRCodePage.razor" },
+        { "tests/ratings", "Pages/Tests/RatingPage.razor" },
+        { "tests/repeater", "Pages/Tests/RepeaterPage.razor" },
+        { "tests/richtextedit", "Pages/Tests/RichTextEditPage.razor" },
+        { "tests/selectlist", "Pages/Tests/SelectListPage.razor" },
+        { "tests/utilities/services", "Pages/Tests/ServicesPage.razor" },
+        { "tests/utilities/shadows", "Pages/Tests/ShadowPage.razor" },
+        { "tests/signaturepad", "Pages/Tests/SignaturePadPage.razor" },
+        { "tests/utilities/sizing", "Pages/Tests/SizingPage.razor" },
+        { "tests/snackbar", "Pages/Tests/SnackbarsPage.razor" },
+        { "tests/spinkit", "Pages/Tests/SpinKitPage.razor" },
+        { "tests/splitter", "Pages/Tests/SplitterPage.razor" },
+        { "tests/steps", "Pages/Tests/StepsPage.razor" },
+        { "tests/tables", "Pages/Tests/TablesPage.razor" },
+        { "tests/tabs", "Pages/Tests/TabsPage.razor" },
+        { "tests/theming", "Pages/Tests/ThemingPage.razor" },
+        { "tests/tests/tooltips", "Pages/Tests/TooltipsPage.razor" },
+        { "tests/treeview", "Pages/Tests/TreeViewPage.razor" },
+        { "tests/typography", "Pages/Tests/TypographyPage.razor" },
+        { "tests/validations", "Pages/Tests/ValidationsPage.razor" },
+        { "tests/vertical-align", "Pages/Tests/VerticalAlignPage.razor" },
+        { "tests/video", "Pages/Tests/VideoPage.razor" },
+    };
+
+    private string gitHubDemoUrl;
+
+    protected override void OnInitialized()
+    {
+        NavigationManager.LocationChanged += OnLocationChanged;
```

Comment:
Add Dispose not to have a memory leak and do the `NavigationManager.LocationChanged -= OnLocationChanged;`